examples/gcp_cli.py

```diff
@@ -163,3 +164,45 @@ def ListServices(args: 'argparse.Namespace') -> None:
   sorted_apis = sorted(results.items(), key=lambda x: x[1], reverse=True)
   for apiname, usage in sorted_apis:
     print('{}: {}'.format(apiname, usage))
+
+
+def GetBucketACLs(args: 'argparse.Namespace') -> None:
+  """Retrieve the Access Controls for a GCS bucket.
+
+  Args:
+    args (argparse.Namespace): Arguments from ArgumentParser.
+  """
+  gcs = gcp_storage.GoogleCloudStorage(args.project)
+  bucket_acls = gcs.GetBucketACLs(args.path)
+  for role in bucket_acls:
+    print('{}: {}'.format(role, ', '.join(bucket_acls[role])))
```

Comment:
Please adjust format string: `{0:s}` (see other sections of the code where this happens)